.gitattributes

```diff
@@ -0,0 +1,15 @@
+# Lock batch file ending
+*.bat text eol=crlf
+
+# Checkout as OS specific line endings, check in as LF
+* text eol=lf
```

Comment:
Based on my reading, this line says...

    Unless I say otherwise, consider all files (of any kind) anywhere in
    the project to be text, and force their line endings to be LF

In other words, this rule seems to undo the previous rule.